Source/DafnyCore/DafnyOptions.cs

```diff
@@ -1146,6 +1147,13 @@ public enum IncludesModes {
       SetZ3Option("type_check", "true");
       SetZ3Option("smt.qi.eager_threshold", "100"); // TODO: try lowering
       SetZ3Option("smt.delay_units", "true");
+      SetZ3Option("model_evaluator.completion", "true");
```

Comment:
Do these Z3 options not significantly change verification behavior?

Reply:
They are on by default in the Language Server already, so I don't think they should.

Reply:
What does it matter whether they're on in the language server? They may still cause different verification behavior on the CLI.

Reply:
These options are meant to only affect the way Z3 processes its counterexample model, so the result of verification will stay the same. I believe @atomb run variability analysis as well, so there are no unexpected performance-related problems. The fact that these options have been used by the Language Server for over two years just means that there are no bugs (that we know of) associated with these options.

Reply:
Sounds good, thanks!